This code doesn't compile 
`[ERROR] public List<WorkflowJob> filterJobs(List<WorkflowJob> jobs, Pattern r) {
[ERROR] ^
[ERROR] /Users/paul/jenkins/pipeline-aggregator-view-plugin/src/main/java/com/ooyala/jenkins/plugins/pipelineaggregatorview/PipelineAggregator.java:196: error: ';' expected
[ERROR] public List<WorkflowJob> filterJobs(List<WorkflowJob> jobs, Pattern r) {
[ERROR] ^
[ERROR] /Users/paul/jenkins/pipeline-aggregator-view-plugin/src/main/java/com/ooyala/jenkins/plugins/pipelineaggregatorview/PipelineAggregator.java:196: error: ';' expected
[ERROR] public List<WorkflowJob> filterJobs(List<WorkflowJob> jobs, Pattern r) {
[ERROR] ^
[ERROR] /Users/paul/jenkins/pipeline-aggregator-view-plugin/src/main/java/com/ooyala/jenkins/plugins/pipelineaggregatorview/PipelineAggregator.java:196: error: not a statement
[ERROR] public List<WorkflowJob> filterJobs(List<WorkflowJob> jobs, Pattern r) {
[ERROR] ^
[ERROR] /Users/paul/jenkins/pipeline-aggregator-view-plugin/src/main/java/com/ooyala/jenkins/plugins/pipelineaggregatorview/PipelineAggregator.java:196: error: ';' expected
[ERROR] public List<WorkflowJob> filterJobs(List<WorkflowJob> jobs, Pattern r) {
[ERROR] ^
[ERROR] /Users/paul/jenkins/pipeline-aggregator-view-plugin/src/main/java/com/ooyala/jenkins/plugins/pipelineaggregatorview/PipelineAggregator.java:272: error: reached end of file while parsing
[ERROR] }`
a missing } might happen when you removed the formatting ? line 193 please check thx

--- a/pom.xml
+++ b/pom.xml
@@ -5,7 +5,7 @@
    <parent>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>plugin</artifactId>
-      <version>2.33</version>
+      <version>2.11</version>
       <relativePath />
    </parent>
 
@@ -15,14 +15,14 @@
    <packaging>hpi</packaging>
 
    <properties>
-      <powermock.version>1.7.3</powermock.version>
+      <powermock.version>1.6.6</powermock.version>
       <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-      <jenkins.version>2.79</jenkins.version>
+      <jenkins.version>2.32.2</jenkins.version>
       <!-- Java Level to use. Java 7 required when using core >= 1.612 -->
-      <java.level>8</java.level>
+      <java.level>7</java.level>
       <!-- Jenkins Test Harness version you use to test the plugin. -->
       <!-- For Jenkins version >= 1.580.1 use JTH 2.x or higher. -->
-      <jenkins-test-harness.version>2.26</jenkins-test-harness.version>
+      <jenkins-test-harness.version>2.15</jenkins-test-harness.version>
       <!-- Other properties you may want to use:
            ~ hpi-plugin.version: The HPI Maven Plugin version used by the plugin..
            ~ stapler-plugin.version: The Stapler Maven plugin version required by the plugin.
@@ -34,7 +34,7 @@
       <dependency>
          <groupId>org.jenkins-ci.plugins.workflow</groupId>
          <artifactId>workflow-job</artifactId>
-         <version>2.14.1</version>
+         <version>2.9</version>
       </dependency>
       <dependency>
          <groupId>junit</groupId>

--- a/src/main/java/com/ooyala/jenkins/plugins/pipelineaggregatorview/PipelineAggregator.java
+++ b/src/main/java/com/ooyala/jenkins/plugins/pipelineaggregatorview/PipelineAggregator.java
@@ -83,9 +83,9 @@ public class PipelineAggregator extends View {
       return useCondensedTables;
    }
 
-   public int getFontSize() {
-      return fontSize;
-   }
+	public boolean isUseScrollingCommits() {
+		return useScrollingCommits;
+	}
 
 	public void setUseScrollingCommits(boolean useScrollingCommits) {
 		this.useScrollingCommits = useScrollingCommits;
@@ -167,29 +167,26 @@ public class PipelineAggregator extends View {
       Pattern r = filterRegex != null ? Pattern.compile(filterRegex) : null;
       List<WorkflowJob> fJobs = filterJobs(jobs, r);
       List<Build> l = new ArrayList();
-		List<WorkflowRun> wfr = new ArrayList<WorkflowRun>();
-		if( !this.onlyLastBuild ) {
-      RunList<WorkflowRun> builds = new RunList(fJobs).limit(buildHistorySize);
-      for ( WorkflowRun build : builds){
-				wfr.add(build);
-			}
-		} else {
-			for(WorkflowJob job : fJobs) {
-				wfr.add(job.getLastBuild());
-			}
-		}
-		if( wfr != null && wfr.size() > 0 ) {
-			for (WorkflowRun build : wfr) {
-         List<ChangeLogSet<? extends ChangeLogSet.Entry>> changeLogSets = ((WorkflowRun) build).getChangeSets();
-         Result result = build.getResult();
-         l.add(new Build(build.getDisplayName(),
-            build.getFullDisplayName(),
-            build.getUrl(),
-            build.getNumber(),
-            build.getStartTimeInMillis(),
-            build.getDuration(),
-            result == null ? "BUILDING" : result.toString(), changeLogSets));
-      }
+	  List<WorkflowRun> wfr = new ArrayList<WorkflowRun>();
+	  if( !this.onlyLastBuild ) {
+		  RunList<WorkflowRun> builds = new RunList(fJobs).limit(buildHistorySize);
+		  for ( WorkflowRun build : builds){
+			  wfr.add(build);
+		  }
+	  } else {
+		  for(WorkflowJob job : fJobs) {
+			  wfr.add(job.getLastBuild());
+		  }
+	  }
+	  if( wfr != null && wfr.size() > 0 ) {
+		  for (WorkflowRun build : wfr) {
+			  List<ChangeLogSet<? extends ChangeLogSet.Entry>> changeLogSets = ((WorkflowRun) build).getChangeSets();
+			  Result result = build.getResult();
+				l.add(new Build(build.getDisplayName(), build.getFullDisplayName(), build.getUrl(), build.getNumber(),
+						build.getStartTimeInMillis(), build.getDuration(), result == null ? "BUILDING" : result.toString(),
+						changeLogSets));
+		  }
+	  }
       return l;
    }
 

